### PR TITLE
Introducing HTC Sync Manager v3.1.87

### PR DESCRIPTION
**Pull request by yurikoles**
I need help with zap stanza, it deletes only files, but not dirs.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
`rmdir` is only for empty directories. If you want to delete directories with files inside, use `delete`.

Also, to solve common stylistic issues (which this cask has), run `brew cask style --fix {{your_cask_file}}`.

File: Casks/htc-sync-manager.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+cask 'htc-sync-manager' do
+  version '3.1.87'
+  sha256 'd0c30eb3ec286d0246c6054f42e6cc0f1a4c1b3cd75530cb851fb0e30e045401'
+
+  url 'http://dl4.htc.com/download/htc-sync-manager/setup_3.1.87_htc.dmg'
+  name 'HTC Sync Manager'
+  homepage 'http://www.htc.com/us/software/htc-sync-manager/'
+  license :gratis # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/htc-sync-manager.rb]: Remove the comment, as it says.

**Comment by yurikoles** [comment]
up&rebase

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 53c705c70e3091d7b4c994dd358959f60a54252f. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**